### PR TITLE
updated nether eye

### PR DESCRIPTION
**Pull request by GildedToxin**
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by GildedToxin** [comment]
This one hurt my soul

File: overrides/config/ftbquests/quests/chapters/hard_eyes.snbt [hunk 1]
```diff
@@ -175,15 +221,42 @@
...
+							"dungeons_libraries:built_in_enchantments": { }
+						}
+						id: "alexsmobs:blood_sac"
+					}
+					type: "item"
+				}
+			]
 			shape: "gear"
-			subtitle: "- Crimson Mosquito Larva: Crafted from a maggot(fly, a common bug in every biome drop), and crimson mosquito probuscis which is obtained by killing a crimson mosquito(in the crimson wastes biome) before it can suck your blood."
+			subtitle: "Crafted with a Maggot and a Crimson Mosquito Probuscis"
```
Comment (RUsami) [inline overrides/config/ftbquests/quests/chapters/hard_eyes.snbt:259]: should be "Crimson Mosquito Proboscis" not "Probuscis" ?

File: overrides/config/ftbquests/quests/chapters/hard_eyes.snbt [hunk 2]
```diff
@@ -271,9 +398,41 @@
 				"213877F5900CC9CE"
 				"5DA4FF9594050625"
 			]
+			description: [
+				"• Potato - Dropped by &cZombies&r or found in by exploring the world"
+				""
+				"• Bread - Crafted with 3 &eWheat&r"
```
Comment (RUsami) [inline overrides/config/ftbquests/quests/chapters/hard_eyes.snbt:404]: Should not Wheat being plural?

**Review by RUsami — CHANGES_REQUESTED** [review]
found 1 error I think, I think Wheat is okay

**Comment by GildedToxin** [comment]
I think wheat is both plural and singular 
but ill change the other one

**Review by Jonah-Hansen — APPROVED** [review]

File: overrides/config/ftbquests/quests/chapters/hard_eyes.snbt [hunk 3]
```diff
@@ -95,7 +95,25 @@
...
+				""
+				"• Wildfire Crown - Crafted with 5 &eWildfire Crown Pieces&r"
+				""
+				"• Mutant Blaze Core - Dropped by &4Mutant Blazes&r"
+				""
+				"• Netherite Scrap - Mined in The Nether depths or found in Bastion chests"
+				""
+				"• Hone: Gild III -  Crafted with weaker &eHone: Gild&r scrolls"
+				""
+				"• Crimson Mosquito Larva - Crafted with a &eMaggot &rand a &eCrimson Mosquito Probuscis&r"
```
Comment (RUsami) [inline overrides/config/ftbquests/quests/chapters/hard_eyes.snbt:109]: There is "Mosquito Proboscis" from "Mosquito Probuscis" here

**Review by RUsami — CHANGES_REQUESTED** [review]
Found 1 spelling error

File: overrides/config/ftbquests/quests/chapters/hard_eyes.snbt (hunk 3 above)
Comment (GildedToxin) [reply overrides/config/ftbquests/quests/chapters/hard_eyes.snbt:109]: AHHH

**Review by RUsami — APPROVED** [review]

**Pull request merged**